Source/BoundaryConditions/PML.cpp

```diff
@@ -760,24 +768,28 @@ PML::PML (const int lev, const BoxArray& grid_ba, const DistributionMapping& gri
         BoxArray grid_cba = grid_ba;
         grid_cba.coarsen(ref_ratio);
 
-        // assuming that the bounding box around grid_cba is a single patch, and not disjoint patches, similar to fine patch.
-        amrex::Box cdomain = grid_cba.minimalBox();
+        BoxArray grid_cba_reduced = grid_cba;
```

Comment:
The code below duplicates the above code. 
Maybe it would be good, in a separate follow-up PR, to create a separate function for this.

Reply:
yes. I ll do this in a separate PR